Glossary
========

-   **Module**: An major component of MIDAS2, including Species, SNVs, CNVs and MIDASDB building.

-   **Analysis**: The single-sample alignment-based tallying of reads mapped to SCGs (species), a representative genome (SNVs), or genes in a pangenome (CNVs) depending on which module is run OR the merging, filtering, and summarization of these results across samples.

-   **Workflow**: The overall conceptual order of events when running MIDAS2 on a dataset, starting from a database (may be implicit) and at least one shotgun metagenomic sample (required) through one or more modules to results that may go into downstream analyses.

-   **Reference database**: The genome assembly data used to generated a MIDAS DB. 

-   **MIDASDB**: The reference database pre-processed into a format to be consumed by the MIDAS2 modules, including marker genes (for species module), represenentative genomes (for SNV module), and pangenomes (for CNV module) for each species in the reference database. Pre-computed MIDASDBs are provided for UHGG and GTDB. Users may customize these  or generate their own MIDASDB from a collection of genome sequences. 

-   **Genome Index**: The data structure that serves as the target to which metagenomic reads are aligned. 

-   **Species-level pangenome**: A set of non-redundant genes (centroids), one for each gene family in each species. Gene families are generated by clustering all gene sequences from all the genomes within each species.
